crmsh/utils.py

```diff
@@ -2287,4 +2249,109 @@ def interface_choice():
     # should consider interface format like "ethx@xxx"
     interface_list = re.findall(r'(?:[0-9]+:) (.*?)(?=: |@.*?: )', out)
     return [nic for nic in interface_list if nic != "lo"]
+
+
+class InterfacesInfo(object):
+    def __init__(self, ipv6=False, second_heartbeat=False, specify_nic_list=[]):
+        self.ipv6 = ipv6
+        self._nic_info_dict = {}
+        self.second_heartbeat = second_heartbeat
+        self.get_interface_info()
```

Comment:
I'm not a big fun of having methods in the `__init__`, but this is just a personal preference hehe
They make unit testing a bit more complicated.
Anyway, it's Ok if you like this way.

Reply:
Changed, no function be called in __init__